app/views/unattended/provisioning_templates/finish/preseed_default_finish.erb

```diff
@@ -60,7 +60,8 @@ echo 'root:<%= root_pass -%>' | /usr/sbin/chpasswd -e
 <%= snippet 'saltstack_setup' %>
 <% end -%>
 
-<%= snippet 'preseed_networking_setup' %>
-<%= snippet 'efibootmgr_netboot' %>
-<%= snippet 'built' %>
+<%= snippet 'preseed_networking_setup' -%>
+<%= snippet 'efibootmgr_netboot' -%>
+<%= snippet 'eject_cdrom' -%>
+<%= snippet 'built' -%>
```

Comment:
https://www.debian.org/releases/stable/amd64/apbs04.en.html#preseed-finish suggests it should do this by default via the regular method. Why is it better to call `eject` manually?

Reply:
A user in the thread linked above reported this was the only workaround that worked. Let me ask explicitly, honestly I haven't tested this.

Reply:
For the record the thread is here: https://community.theforeman.org/t/failed-to-detach-iso-image-from-cdrom-drive-from-vmware/21506/14

Reply:
Yeah it seems that vmware blocks unmount of the image, if the cdrom is still in the mechanic.
So this aligns it with baremetal provisioning and with other provisioning methods (kickstart, autoyast).